src/T2DMap.cpp

```diff
@@ -4032,15 +4032,26 @@ void T2DMap::slot_setArea()
         return;
     }
 
-    QMapIterator<int, QString> it(mpMap->mpRoomDB->getAreaNamesMap());
-    while (it.hasNext()) {
-        it.next();
-        int areaID = it.key();
-        if (areaID > 0) {
-            arealist_combobox->addItem(QStringLiteral("%1 (%2)").arg(it.value(), QString::number(areaID)), QVariant(areaID));
-        }
+    QStringList sortedAreaList;
+    sortedAreaList = mpMap->mpRoomDB->getAreaNamesMap().values();
+
+    QCollator sorter;
+    sorter.setNumericMode( true ); // Default is false
```

Comment:
```suggestion
    sorter.setNumericMode(true);
```

Don't need to mention what the defaults are, doesn't do much